apps/mull-api/src/app/app.module.ts

```diff
@@ -49,6 +49,11 @@ import { AuthModule } from './auth/auth.module';
       uploads: {
         maxFileSize: 10000000000000,
       },
+      cors: {
+        origin: environment.client.baseUrl,
```

Comment:
Think we can add [Apollo Studio](https://studio.apollographql.com/) during dev only? We would add https://studio.apollographql.com/ to the `origin` property here and in `main.ts`